### PR TITLE
TB Form WIP

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

File: configuration/pih/htmlforms/tbIntake.xml [hunk 1]
```diff
@@ -1,5 +1,344 @@
...
+            jq('#calculated-edd-and-gestational').hide();
+
+            jq("#lastPeriodDate input[type='hidden']").change(function() {
+            updateEdd();
+            });
+
+            updateEdd();
+            });
+        </script>
+    </ifMode>
```
Comment (brandones) [inline configuration/pih/htmlforms/tbIntake.xml:151]: Why all this? Doesn't look like there's anything about pregnancy specified or implemented in this form.

File: configuration/pih/htmlforms/tbIntake.xml [hunk 2]
```diff
@@ -1,5 +1,344 @@
...
+
+        .section {
+            width: 100%;
+        }
+
+        .two-columns {
+            column-count: 2;
+            -webkit-column-count: 2;
+            -moz-column-count: 2;
+        }
```
Comment (brandones) [inline configuration/pih/htmlforms/tbIntake.xml:68]: If you're going to use a two-column format, please use

```suggestion
            /* A div with class="two-columns" must have exactly two children divs */
            .two-columns {
                display: table;
                height: 100%;
                width: 100%;
            }

            .two-columns > div {
                display: table-cell;
                width: 50%;
            }
```

**Comment by brandones** [comment]
Looks good other than those two notes. Our norm is to aggressively message code every UI string, but I think it's totally reasonable to wait to do that until someone actually expresses interest in using this form in another language.

**Comment by mogoodrich** [comment]
@brandones @MiguelAHPpih  yeah, I'm on board with just keeping it in Spanish for now....

File: configuration/pih/htmlforms/tbIntake.xml [hunk 3]
```diff
@@ -1,5 +1,297 @@
...
+        </p>
+        <p><uimessage code="pih.form.last_updated_on"/>:
+            <lookup complexExpression="$form.dateChanged"/>
+        </p>
+        <p><uimessage code="pih.form.printed_on"/>:
+            <lookup complexExpression="$formGeneratedDatetime"/>
+        </p>
+    </div>
+
+    <section id="bk-test" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.tbintake.smear">
```
Comment (brandones) [inline configuration/pih/htmlforms/tbIntake.xml:138]: No need to add `id`s if there's nothing referring to them.

File: configuration/pih/htmlforms/tbIntake.xml [hunk 4]
```diff
@@ -1,5 +1,297 @@
...
+
+    <section id="comorbity-hiv" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.tbintake.comorbityHIV">
+        <div class="section-container">
+            <label>
+                <uimessage code="¿Presenta comorbilidad de VIH?"/>
+            </label>
+            <p>
+                <obs conceptId="CIEL:138405"
+                     answerConceptIds="CIEL:1065,CIEL:1066"
+                     answerLabels="Si,No"
```
Comment (brandones) [inline configuration/pih/htmlforms/tbIntake.xml:268]:
```suggestion
                     answerLabels="Sí,No"
```

**Review by brandones — APPROVED** [review]
Looks good!

**Pull request merged**